CONTRIBUTING.adoc

```diff
@@ -0,0 +1,152 @@
+ ** Alternatively you may use raw Vulkan API calls.
+* Code and assets:
+ ** Single source file samples with minimal build complexity are encouraged to make porting to different platforms easier.
+ ** Compiling the sample with the highest warning level and warnings-as-errors (-Wall -Wextra -Werror, or /Wall /WX) is highly recommended.
+ ** Shaders are saved in the `/shaders/` folder, in a separate sub-folder with the same name as the sample sub-folder in `/samples/api/`, `/samples/performance/` or `/samples/extensions/`.
+ ** Assets should be saved in https://github.com/KhronosGroup/Vulkan-Samples-Assets[vulkan-samples-assets].
+* License:
+ ** Samples are licensed under the link:LICENSE[LICENSE] file in the root folder.
+ ** The current Contributor License Agreement (CLA) only allows samples to be licensed under the Apache 2.0 license.
+ ** Every source code file must have a Copyright notice and license at the top of the file as described below.
+ ** Assets must have their own license.
+* Third party libraries:
+ ** A sample may not depend on a separate installation of a third party library.
+ ** Any third party library that is used needs to be available under a compatible open source license i.e.
+MIT or Apache 2.0.
+ ** Any third party library that is used must be included as a sub-module in the `/third_party/` folder.
+
+== Copyright Notice and License Template
+
+To apply the Apache 2.0 License to your work, attach the following boilerplate notice, with the fields enclosed by brackets "[]" replaced with your own identifying information for the copyright year or years.
+_Don't include the brackets!_ The text should be enclosed in the appropriate comment syntax for the file format.
+We also recommend that a file or class name and description of purpose be included on the same "printed page" as the copyright notice for easier identification within third-party archives.
+When contributing to an existing file you may add a new copyright year line under the existing ones.
+
+....
+Copyright [yyyy] [name of copyright owner]
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+....
+
+== Code Style
+
+A common code style like, for instance, the one described by the https://google.github.io/styleguide/cppguide.html[Google C{pp} Style Guide] is recommended.
+A sample must consistently apply a single code style.
+
+A `.clang-format` file is included with this repository, please use `clang-format -style=file` to verify the code style.
+Ensure that you are using clang-format 8.
```

Comment:
Is clang-format 8 still the correct version?

Reply:
Not sure. This is just a conversion (and some minor fixes) of the current documentation. If this is outdated it would be best to fix it with the current markdown and I'll merge that change.

Reply:
15 is the correct version. We should do a documentation pass separate to this PR